packages/design-system/src/components/FieldContainer/FieldContainer.tsx

```diff
@@ -105,7 +106,7 @@ export class FieldContainer extends React.Component<FieldContainerProps> {
       className,
       component,
       errorMessage,
-      errorPlacement,
+      errorPlacement = errorPlacementDefault(),
```

Comment:
The flag is used by form components in `FieldContainer`. Dropdown and TextField are the exceptions, they also use the flag because they are also needed inside the form input components (i.e. Select and TextInput)